SparkPost/2web2ui
# SA-1681: Fixed issue with parsing old filters

### What Changed
 - Parses old filters

### How To Test
 - On staging, appteam, go to `http://app.sparkpost.test/signals/analytics?from=2020-10-02T15%3A00%3A00Z&to=2020-10-09T15%3A36%3A39Z&range=7days&timezone=America%2FNew_York&precision=hour&metrics=count_sent&metrics=count_unique_confirmed_opened_approx&metrics=count_accepted&metrics=count_bounce&filters=Recipient%20Domain%3Agmail.com&filters=Sending%20Domain%3Asparkpostbox.com&filters=Subaccount%3ASparkle%20Ponies%20%28ID%20123%29%3A123&report=5e5ee8e3-8e5d-42a9-a0b9-bb3ea3eadea0`
 - Enable account UI feature flag `allow_report_filters_v2`
- Make sure that the filters for properly populates

### To Do
